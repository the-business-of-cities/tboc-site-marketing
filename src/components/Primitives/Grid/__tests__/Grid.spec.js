import { Grid } from "../index";
import React from "react";
import { shallow } from 'enzyme';

describe("Grid", () => {
  it("renders correctly", () => {
    expect(shallow(<Grid />)).toMatchSnapshot();
  });
});
