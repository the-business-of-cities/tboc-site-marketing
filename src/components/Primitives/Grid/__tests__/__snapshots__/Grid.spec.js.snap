// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid renders correctly 1`] = `
.c0 {
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 3em;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

@media (max-width:999px) {
  .c0 {
    -webkit-flex-basis: 100%;
    -ms-flex-preferred-size: 100%;
    flex-basis: 100%;
    padding: 1em;
  }
}

<div
  className="c0"
/>
`;
